pkg/apps/apiserver/registry/rollback/rest.go

```diff
@@ -106,7 +106,12 @@ func (r *REST) Create(ctx context.Context, obj runtime.Object, createValidation
 		from.Annotations[key] = value
 	}
 
-	return r.generator.GenerateRollback(from, toInternal, &rollback.Spec)
+	fromInternal := &appsapi.DeploymentConfig{}
+	if err := legacyscheme.Scheme.Convert(from, fromInternal, nil); err != nil {
```

Comment:
given you have switched all REST and it is the only caller to `GenerateRollback` would it make sense to switch that method as well, convert `rollback.Spec` to external and drop the 2 conversion to internal? could be a follow up if you want

Reply:
follow up is fine this time :-)